config/initializers/rack_cors.rb

```diff
@@ -0,0 +1,13 @@
+module App
+  class Application < Rails::Application
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*',
+                 headers: :any,
+                 methods: %i[get post options put delete],
+                 expose: %w[access-token uid client]
```

Comment:
Just curious, why are this changes needed?

Reply:
i dont need it!